meta-ibm/recipes-phosphor/webui/webui-vue/change-logo.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Start the change logo script
+
```

Comment:
Lets add a "After=basic.target" and a "Before=bmcweb.service" to be sure the basics are up and going and we get this done before bmcweb starts.

Reply:
Done